ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -211,10 +211,11 @@ enum CocoaPodUtils {
 
     for (podName, version) in pods {
       let subspecArray = podName.components(separatedBy: "/")
-      if subspecArray.count > 2 {
+      if subspecArray.count == 1 || subspecArray[0] == "abseil" {
+        // Special case for abseil since it has two layers and no external deps.
+        versions[subspecArray[0]] = version
+      } else if subspecArray.count > 2 {
         fatalError("Multi-layered subspecs are not supported - \(podName)")
-      } else if subspecArray.count == 1 {
```

Comment:
Was this `else` statement just for abseil before?

Reply:
abseil was falling into the error case before since it has a two-layered subspec. Now we still error out on two-layer subspecs except for abseil.